src/__tests__/annotations.test.ts

```diff
@@ -281,27 +282,41 @@ describe('Annotations Query', () => {
         query: "events{assetIds=[$AssetVariable], description!~'event.*'}",
       },
     };
-    const response = _.cloneDeep(annotationResponse);
-    response.data.items = annotationResponse.data.items.filter(item =>
+    const annotationOption2: any = {
+      ...annotationOption1,
+      annotation: {
+        query: 'events{assetIds=[${MultiValue:csv}]}',
+      },
+    };
+    const response1 = _.cloneDeep(annotationResponse);
+    const response2 = _.cloneDeep(annotationResponse);
+
+    response1.data.items = annotationResponse.data.items.filter(item =>
       item.assetIds.some(id => id === 123)
     );
+    response2.data.items = annotationResponse.data.items.filter(
+      item => item.assetIds.some(id => id === 123 || id === 456) // see $MultiValue definition
+    );
```

Comment:
what do we test here? maybe we could get rid of logic and do something like
`response2.data.items = [annotationResponseItem123, annotationResponseItem123_456]`
?
same above

Reply:
here we're filtering through the mocked response and getting expected results for queries. Response items are repeated for response 1 and response 2 (means that response 1 and 2 has few same items), so decided here to filter like this

Reply:
but overall, this can be removed, cause here the main purpose to check request filters